client/src/utils/userInfo.js

```diff
@@ -1,8 +1,10 @@
 export const getInitials = (firstName, lastName) => {
   return firstName.charAt(0).toUpperCase() + lastName.charAt(0).toUpperCase();
 };
-
 export const getInitialsFromFullName = (fullName) => {
   const [firstName, lastName] = fullName.split(" ");
   return firstName.charAt(0).toUpperCase() + lastName.charAt(0).toUpperCase();
 };
+export const getOrgInitials = (firstName) => {
+  return firstName.charAt(0).toUpperCase()
+};
```

Comment:
```suggestion
export const getOrgInitials = (fullOrgName) => {
  const orgNameSplit = fullOrgName.split(" ");
  return orgNameSplit[0].charAt(0).toUpperCase() + (orgNameSplit[1]) ? orgNameSplit[1].charAt(0).toUpperCase() : ""; 
};
```

Reply:
Or some logic like this one

Reply:
That change can be integrated in `getInitialsFromFullName` function and it doesn't require a new one.

Reply:
That's truee! I think we should make the body of that funtion more generic, so not first last name and first name. What do you think @silenceway ?